spec/attribute.dd

```diff
@@ -317,6 +317,17 @@ $(GNAME DeprecatedAttribute):
         oldBar();   // Deprecated: function test.oldBar is deprecated - Don't use bar
         ---------------
 
+        $(P While direct function call are not allowed, manifest constants are.
```

Comment:
With current master, direct function calls are *allowed*. The change is introduced while reviewing https://github.com/D-Programming-Language/dmd/pull/5302 .

I'm asking why the behavior is necessary in:
https://github.com/D-Programming-Language/dmd/pull/5302#issuecomment-182713890